impl/client/src/main/javascript/web/pentaho/type/action/ListChangeset.js

```diff
@@ -125,12 +125,78 @@ define([
      *
      * @param {string} key - The key of the element.
      *
-     * @return {pentaho.type.action.ComplexChangeset} The child changeset or `null`.
+     * @return {pentaho.type.action.Changeset} The child changeset or `null`.
```

Comment:
Actually, because lists cannot contain other lists, this can only be a `ComplexChangeset` :-)